target/linux/ipq806x/files-6.6/arch/arm/boot/dts/qcom/qcom-ipq8064-wpq864.dts

```diff
@@ -404,8 +411,99 @@
 	pinctrl-names = "default";
 };
 
-&flash {
-	compatible = "jedec,spi-nor";
+&gsbi5 {
```

Comment:
Why do you extend the `gsbi5` node and not the `flash` node?

When you extend the `flash` node you have to delete the `partition@1` node like this:
```
/delete-node/ partition@1;
```
https://elixir.bootlin.com/linux/v6.6.52/source/arch/arm/boot/dts/qcom/qcom-ipq8064-v1.0.dtsi#L51

Reply:
THX for review.
I check gsbi5 vs flash.
Should I add the BUG #15476 Description in the commit comment?

Reply:
yes please add a description why you change the code into the commit comment.

Reply:
@hauke 
I have change the extend from gsbi5 node to the flash node and delete the partition@1 node. Is tested an working fine. :-)